src/db.rs

```diff
@@ -360,6 +361,18 @@ impl Db {
         Self::open_with_opts(path, DbOptions::default(), object_store).await
     }
 
+    pub async fn open_with_cache(
```

Comment:
why do we need this method - can't you just specify opts if you want to use your own cache?

Reply:
I've removed it.